src/NuGet.Clients/NuGet.SolutionRestoreManager/ErrorListTableEntry.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Shell.TableControl;
+using Microsoft.VisualStudio.Shell.TableManager;
+using NuGet.Common;
+
+namespace NuGet.SolutionRestoreManager
+{
+    internal class ErrorListTableEntry : ITableEntry
+    {
+        internal const string ErrorSouce = "NuGet";
+
+        public object Identity
+        {
+            get
+            {
+                return Message;
+            }
+        }
+
+        public string Message { get; set; }
+
+        public LogLevel LogLevel { get; set; }
+
+        public ErrorListTableEntry(string message, LogLevel level)
+        {
+            Message = message;
+            LogLevel = level;
+        }
+
+        public bool CanSetValue(string keyName)
+        {
+            return false;
+        }
+
+        public bool TryGetValue(string keyName, out object content)
+        {
+            content = null;
+
+            switch (keyName)
+            {
+                case StandardTableColumnDefinitions.Text:
```

Comment:
it would be nice if it also support HelpLink so that user can see how to resolve the issue.